src/signals/incident/definitions/wizard-step-5-samenvatting.js

```diff
@@ -389,6 +389,200 @@ export default {
+      },
+      extra_wonen_leegstand_iemand_aanwezig: {
+        label: 'Iemand aanwezig',
+        render: PreviewComponents.PlainText,
+        optional: true,
+      },
+
+      // wonen - leegstand en onderhuur
+      extra_wonen_iemand_aanwezig: {
+        label: 'Iemand aanwezig',
+        render: PreviewComponents.PlainText,
+        optional: true,
+      },
+
+      // wonen - vakantieverhuur
+      extra_wonen_vakantieverhuur_toeristen_aanwezig: {
+        label: 'Toeristen aanwezig',
+        render: PreviewComponents.ObjectValue,
+        optional: true,
+      },
+      extra_wonen_vakantieverhuur_bellen_of_formulier: {
+        label: 'Bellen of meldingsformulier',
+        render: PreviewComponents.ObjectValue,
+        optional: true,
+      },
+      extra_wonen_vakantieverhuur_aantal_mensen: {
+        label: 'Aantal personen',
+        render: PreviewComponents.ObjectValue,
+        optional: true,
+      },
+      extra_wonen_vakantieverhuur_hoe_vaak: {
+        label: 'Hoe vaak',
+        render: PreviewComponents.ObjectValue,
+        optional: true,
+      },
+      extra_wonen_vakantieverhuur_wanneer: {
+        label: 'Wanneer',
+        render: PreviewComponents.ObjectValue,
+        optional: true,
+      },
+      extra_wonen_vakantieverhuur_bewoning: {
+        label: 'Bewoning',
+        render: PreviewComponents.ObjectValue,
+        optional: true,
+      },
+      extra_wonen_vakantieverhuur_naam_bewoner: {
+        label: 'Naam bewoner',
+        render: PreviewComponents.PlainText,
+        optional: true,
+      },
+      extra_wonen_vakantieverhuur_online_aangeboden: {
+        label: 'Online aangeboden',
+        render: PreviewComponents.ObjectValue,
+        optional: true,
+      },
+      extra_wonen_vakantieverhuur_link_advertentie: {
+        label: 'Link advertentie',
+        render: PreviewComponents.PlainText,
+        optional: true,
+      },
+
+      // wonen - woonkwaliteit
+      extra_wonen_woonkwaliteit_direct_gevaar: {
+        label: 'Direct gevaar',
+        render: PreviewComponents.ObjectValue,
+        optional: true,
+      },
+      extra_wonen_woonkwaliteit_gemeld_bij_eigenaar: {
+        label: 'Gemeld bij eigenaar',
+        render: PreviewComponents.ObjectValue,
+        optional: true,
+      },
+      extra_wonen_woonkwaliteit_bewoner: {
+        label: 'Bewoner',
+        render: PreviewComponents.ObjectValue,
+        optional: true,
+      },
+      extra_wonen_woonkwaliteit_namens_bewoner: {
+        label: 'Namens bewoner',
+        render: PreviewComponents.ObjectValue,
+        optional: true,
+      },
+      extra_wonen_woonkwaliteit_toestemming_contact: {
+        label: 'Toestemming contact opnemen',
+        render: PreviewComponents.ObjectValue,
+        optional: true,
+      },
+      extra_wonen_woonkwaliteit_geen_contact: {
+        label: 'Toestemming contact opnemen',
+        render: PreviewComponents.PlainText,
+        optional: true,
```

Comment:
Overal lijkt `optional` 'true' te zijn. Is het wel nodig om die prop te definiëren?

Reply:
bijna alle zijn true. dat klopt dat moet eigenlijk andersom: zal ik dat nu veranderen? of wen andere pr maken?